README.md

```diff
@@ -110,16 +111,19 @@ Lodash template function does not provide `include` method of [ejs module](http:
 
 As a result, `renderedHtml` becomes a string `<h1><a href="http://example.com">Example</a></h1>`.
 
-
-
 ## Release History
-* 1.0.0 - Changed export format to ESM
+* 1.0.0-beta.1 - Inital fork of [ejs-loader](https://github.com/difelice/ejs-loader) with support for ECMA Modules and deprecating CommonsJS
```

Comment:
Unless you're anticipating issues, I would skip the beta tag. Otherwise, we lose out on automatic patches and will have to have corresponding update PRs in grove core for minor bug fixes.

Reply:
@racoleman this is just to test the package out. Will bump to 1.0.0 so we can get patch support once ready